service/src/test/java/bio/terra/workspace/service/workspace/flight/gcp/DeleteGcpContextFlightTest.java

```diff
@@ -181,22 +181,4 @@ void deleteContextUndo() throws Exception {
     // Because this flight cannot be undone, the context should still be deleted even after undoing.
     assertTrue(gcpCloudContextService.getGcpCloudContext(workspaceUuid).isEmpty());
   }
-
-  @Test
-  void deleteNonExistentContextIsOk() throws Exception {
```

Comment:
Because I couldn't think of a reason why we should hide the not-found error. It was not hidden for the Azure case.
Do you know a reason we should hide this from callers?